rwengine/src/audio/SoundManager.hpp

```diff
@@ -26,14 +28,14 @@
 /// instances simultaneously without duplicating raw source).
 class SoundManager {
 public:
-    SoundManager();
+    SoundManager(GameData* _data);
```

Comment:
Member variables should start with an underscore.

Reply:
Which version do you prefer for member `_data`, `m_data`, `data_`?

Reply:
`_data`